make_a_plea/management/commands/zendesk.py

```diff
@@ -0,0 +1,212 @@
+
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '--to-csv',
+            action='store_true',
+            dest='to-csv',
+            default=False,
+            help='Export all relevant zendesk items to csv file'),
+        )
+
+    def handle(self, *args, **options):
+        self.load_settings()
+        self.init_zen()
+
+        if options["to-csv"]:
+            self.to_csv()
+
+    def load_settings(self):
+        with open(os.path.expanduser("~/.hmcts-zendesk"), "r") as config_file:
+            self.settings = yaml.load(config_file.read())
+
+    def init_csv(self, f, dialect=csv.excel):
+        self.writer = csv.writer(f, dialect=dialect)
+
+    def init_zen(self):
+        self.zendesk = Zendesk(
+            self.settings["url"],
+            self.settings["email"],
+            self.settings["password"])
+
+    def get_all_fieldnames(self):
+        return \
+            SIMPLE_FIELDS + \
+            NEWLINE_FIELDS + \
+            STRUCTURED_FIELDS
+
+    def to_csv(self):
+        self.outfile = self.settings.get('outfile', 'zendump.csv')
+        self.batch_size = 100
+        with open(self.outfile, "wb") as f:
+
+            self.init_csv(f)
+            self.writerow(self.get_all_fieldnames())  # CSV Header
+
+            for batch in self.yield_ticket_batches():
+                for ticket in batch:
+                    self.writerow(self.ticket_to_row(ticket))
+
+    def writerow(self, row):
+        self.writer.writerow([
+            cell.encode("utf-8")
+            for cell in row])
+
+    def ticket_to_row(self, ticket):
+        return \
+            self.get_simple_fields(ticket) + \
+            self.get_newline_fields(ticket) + \
+            self.get_structured_fields(ticket)
+
+    def parse_simple_field(self, field):
+        try:
+            return field.encode("utf-8")
+        except AttributeError:  # ints
+            return str(field).encode("utf-8")
+
+    def get_simple_fields(self, ticket):
+        fields = []
+        for field in SIMPLE_FIELDS:
+            if field in ticket:
+                fields.append(self.parse_simple_field(ticket[field]))
+            else:
+                fields.append("")
+        return fields
+
+    def parse_newline_field(self, field):
+        return field or ""
+
+    def get_newline_fields(self, ticket):
+        fields = []
+        for field in NEWLINE_FIELDS:
+            if field in ticket and field is not None:
+                fields.append(self.parse_newline_field(ticket[field]))
+            else:
+                fields.append("")
+        return fields
+
+    def parse_structured_field(self, field):
+        return str(field)
+
+    def get_structured_fields(self, ticket):
+        fields = []
+        for field in STRUCTURED_FIELDS:
+            if field in ticket:
+                fields.append(self.parse_structured_field(ticket[field]))
+            else:
+                fields.append("")
+        return fields
```

Comment:
If `parse_structured_field` were updated to handle `None` this could be
```python
return [self.parse_structured_field(ticket.get(field)) for field in STRUCTURED_FIELD]
```